src/main/java/appeng/tile/AEBaseTileEntity.java

```diff
@@ -434,7 +424,6 @@ public boolean hasCustomInventoryName() {
 
     public void securityBreak() {
         this.world.destroyBlock(this.pos, true);
-        this.disableDrops();
```

Comment:
The whole method is currently unused. This needs to be investigated in case we did accidentally break it and it still needs to ability to disabled drops.

Reply:
`securityBreak` is used for non-part tiles such as full-block interfaces. I tested this with interfaces containing an inventory and it's working fine. There is really no reason to disable the drops because the block is already destroyed.

Fyi this was added in the original security commit in 2014, it's probably not relevant anymore assuming it ever was to begin with.

Reply:
Now that #5127 is merged, I need to investigate if this function is useful to have at all.

Reply:
This implements securityBreak for IGridHost, even though AEBaseTileEntity does not implement IGridHost. Don't ask.... other tile entities that implement IGridHost implement that function by inheriting it from AEBaseTile -> blergh.

Reply:
I documented why this method exists since it's not obvious, and thus it should be fine.

Reply:
Yup, looks good.

Reply:
This is still not resolved. The whole code path no longer exists and might result in unexpected behaviour. E.g. there might be a special case in which a player is able to pickup a drive with its cells and it then suddenly drops the whole drive with cells included plus additionally the cells itself again. I just have no idea how to test it currently. Or in which cases it might actually apply (and has to).

Reply:
> The whole code path no longer exists.

Then how is securityBreak() handled for tiles such as a drive?

> I just have no idea how to test it currently. Or in which cases it might actually apply (and has to).

That's what alphas are for.